### ISSUE TITLE
Check to ensure newer app version adoption is correlating to less requests during maintenance windows.

### ISSUE THREAD

**Issue by aherzberg** [department-of-veterans-affairs/va-mobile-app#8875]
See previous ticket at https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8841. 

Wait to play this until enough time has passed for app 2.28.0+ has been the vast majority of active users.

**Comment by jperk51** [comment]
@aherzberg do you know if the FE has started sending the version with requests vs just the build number?

**Comment by kpethtel** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/VA-Mobile-60f1a34998bc75000f2a489f/issues/department-of-veterans-affairs/va-mobile-app/8875?planning-poker) with Zenhub @aherzberg @jperk51 @Tonksthebear

**Comment by aherzberg** [comment]
nah, looks like it got pointed and put in backlog https://app.zenhub.com/workspaces/va-mobile-60f1a34998bc75000f2a489f/issues/gh/department-of-veterans-affairs/va-mobile-app/8661

**Comment by jperk51** [comment]
Ight @StacyB2023 this is blocked until the FE starts passing the app version with requests.

**Comment by BeccaTupaj** [comment]
Closing due to this ticket not including information from original zenhub tickets.
